pkg/agent/stream/resourceclient.go

```diff
@@ -65,3 +66,28 @@ func (c *resourceClient) get(selfLink string) (*apiv1.ResourceInstance, error) {
 
 	return ri, err
 }
+
+// func (c *resourceClient) create(r *apiv1.ResourceInstance) (*apiv1.ResourceInstance, error) {
+// 	token, err := c.auth.GetToken()
+// 	if err != nil {
+// 		return nil, err
+// 	}
+//
+// 	req := api.Request{
+// 		Method:  http.MethodPost,
+// 		URL:     fmt.Sprintf("%s/%s/%s/%s/%s", c.url, r.Group, r.APIVersion, r.Kind, r.Name),
+// 		Headers: make(map[string]string),
+// 	}
+//
+// 	req.Headers["Authorization"] = "Bearer " + token
+// 	req.Headers["X-Axway-Tenant-Id"] = c.tenantID
+// 	req.Headers["Content-Type"] = "application/json"
+// 	res, err := c.client.Send(req)
+// 	if err != nil {
+// 		return nil, err
+// 	}
+//
+// 	err = r.UnmarshalJSON(res.Body)
+//
+// 	return r, err
+// }
```

Comment:
just dead for now?

Reply:
I'll come back to this. I started some work for a different branch on here too.